packages/strapi-design-system/src/Select/SelectButton.js

```diff
@@ -53,11 +53,13 @@ export const SelectButton = forwardRef(({ labelledBy, expanded, onTrigger, disab
   return (
     <StyledSelectButton
       ref={ref}
+      role="combobox"
       aria-labelledby={labelledBy}
       aria-haspopup="listbox"
       aria-expanded={expanded}
       onKeyDown={handleKeyDown}
-      aria-disabled={disabled}
+      disabled={disabled}
```

Comment:
Ok ... if you think this is fine I will trust you on that because I don't have the time to manually test this today. Feel free to resolve the thread if you don't see a problem.

Reply:
I'm basing these assumptions off how other libraries have implemented this and my own research. So while I think it should be fine, we can always revert it if we face issues 👍🏼